lib/jxl/simd_util-inl.h

```diff
@@ -0,0 +1,257 @@
+// Copyright (c) the JPEG XL Project Authors. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Misc utilities for SIMD operations
```

Comment:
That's a quite generic description for what is currently only specifically interleaving.

I suppose more could be added later; possibly there are already existing "simd util" things that could be moved here?